src/test/resources/com/puppycrawl/tools/checkstyle/checks/header/regexpheader/InputRegexpSmallHeader.java

```diff
@@ -0,0 +1,6 @@
+package com.puppycrawl.tools.checkstyle.checks.header.regexpheader;
+
+
+/**
+ */
+public class InputRegexpSmallHeader {}
```

Comment:
So I should name 'InputRegexpSmallHeader' to 'InputRegexpHeader'? that would create a conflict there is already a class named InputRegexpHeader.